### PR TITLE
Added examples for CG utils in README

### PR DESCRIPTION
**Pull request by mchakra2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -336,6 +336,71 @@ print(variables)
...
+...
+
+```
+
+### Sparse Mapping
+
+The `sparse_mapping(...)` method creates the necessary indices and values for defining a sparse tensor in tensorflow that is a mass-weighted MxN mapping operator where M is the number of coarse-grained particles and N is the number of atoms in the system.
+
+```python
+# Use in the model python file. The example is shown for 1 coarse-grained site per molecule.
```
Comment (whitead) [inline README.md]: We no longer separate into model/run files. This code requires the hoomd system, so it definitely makes sense to not have this in a separate file

File: README.md [hunk 2]
```diff
@@ -336,6 +336,71 @@ print(variables)
...
+...
+molecule_mapping_matrix = numpy.ones([1, len(molecule_mapping_index[0])], dtype=np.int)
+mapping_per_molecule = [molecule_mapping_matrix for _ in molecule_mapping_index]
+cg_mapping = hoomd.tensorflow_plugin.sparse_mapping(mapping_per_molecule, molecule_mapping_index, system = system)
+...
+'''
+mapping_per_molecule:
```
Comment (whitead) [inline README.md]: This doesn't keep its formatting when rendered.

File: README.md [hunk 3]
```diff
@@ -336,6 +336,71 @@ print(variables)
...
+```
+
+### Sparse Mapping
+
+The `sparse_mapping(...)` method creates the necessary indices and values for defining a sparse tensor in tensorflow that is a mass-weighted MxN mapping operator where M is the number of coarse-grained particles and N is the number of atoms in the system.
+
+```python
+# Use in the model python file. The example is shown for 1 coarse-grained site per molecule.
+...
+molecule_mapping_matrix = numpy.ones([1, len(molecule_mapping_index[0])], dtype=np.int)
```
Comment (whitead) [inline README.md]: Add some comments so people can understand what is happening here

File: README.md [hunk 4]
```diff
@@ -336,6 +336,71 @@ print(variables)
...
+...
+mapped_position = hoomd.tensorflow_plugin.center_of_mass(graph.positions[:,:3], cg_mapping, system)
+...
+'''
+cg_mapping:
+The output from the sparse_matrix(...) method output indicating how each molecule is mapped into coarse-grained sites.
+'''
+```
+
+### Compute Nlist
```
Comment (whitead) [inline README.md]: Capitalization

File: README.md [hunk 5]
```diff
@@ -336,6 +336,71 @@ print(variables)
...
+'''
+mapped_position:
+Mapped particle positions obeying the periodic boundary condition as returned by the center_of_mass(...) method.
+rcut: The cut of radius
+NN : Number of nearest neighbors to be considered for the coarse-grained system
+'''
+```
+
+
 TODO: Separate into two files and document the functions.
```
Comment (whitead) [inline README.md]: Remove these

File: README.md (hunk 2 above)
Comment (hgandhi2411) [reply README.md]: It renders correctly for me.

Comment (whitead) [reply README.md]: Notice how it doesn't wrap because it's treated as code:

Screenshot from 2019-04-22 12-20-14

Comment (hgandhi2411) [reply README.md]: Ah! I see. In my editor that text is wrapped so I didn't see that.

File: README.md [hunk 6]
```diff
@@ -336,9 +336,52 @@ print(variables)
...
+
+...
+mapped_position = htf.center_of_mass(graph.positions[:,:3], cg_mapping, system)
+#cg_mapping is the output from the sparse_matrix(...) method and indicates how each molecule is mapped.
+...
+
+```
+
+### Compute Mapped Neighbor List
+The `compute_nlist(...)` method returns the neighbor list for the mapped coarse-grained particles. In the example, mapped_position is the mapped particle positions obeying the periodic boundary condition as returned by the `center_of_mass(...) method`, `rcut` is the cut-off radius and `NN` is the number of nearest neighbors to be considered for the coarse-grained system.
```
Comment (hgandhi2411) [inline README.md]: Put mapped_position in ` quotes`  to be consistent. All variable names are formatted as code.

**Pull request merged**